### PR TITLE
Fixes for /sub-pages

### PR DESCRIPTION
**Pull request by sebworks**
- Added body_content block
- Added content_main modifier
- Stacked css classes
- Added check for empty sub_pages.

## Changes
- Updated  'sub-pages/_single.html' to add content block and add minor tweaks.

## Test
- Visit 'http://localhost:7000/sub-pages/file-a-privacy-complaint/'

## Review 
- @jimmynotjim  
- @anselmbradford  
- @KimberlyMunoz 
- @kurtw

### CODE REVIEW

File: sub-pages/_single.html [hunk 1]
```diff
@@ -20,17 +21,25 @@
...
     {% endif %}
 
-    {% if vars.sub_pages %}
-    <section class="block block__padded-top block__border-top block__flush-bottom">
+    {% if sub_pages and sub_pages.total %}
+    <section class="block
+                    block__padded-top
+                    block__border-top
+                    block__flush-bottom
+                    block__border-bottom">
```
Comment (anselmbradford) [inline sub-pages/_single.html]: `block__border-bottom` was added, is that intentional?

**Comment by anselmbradford** [comment]
Would it make sense to remove or rename `initiative_topic` and `initiative_description`?

File: sub-pages/_single.html (hunk 1 above)
Comment (sebworks) [inline sub-pages/_single.html]: It should be removed

**Comment by sebworks** [comment]
I would say let's rename them for now. We may find we don't need them and remove them in the future. Any suggestions on the names?

**Comment by KimberlyMunoz** [comment]
Looks good. 

Do we have a page where a page has a subpage to test L31-71?

**Comment by sebworks** [comment]
@KimberlyMunoz , you have test it on the /sub-pages/file-a-privacy-complaint/ page.

**Comment by KimberlyMunoz** [comment]
Hashed it out in chat. :+1:

**Pull request merged**